### DISCUSSION TITLE
Problem with StarStruck Badge

### DISCUSSION THREAD

**Discussion by Crushoverride007** [Schweinepriester/github-profile-achievements#199] [Q&A] [answered]
even though i've a 16 stars on a repo, still the badge not showing up?
<img width="667" alt="image" src="https://user-images.githubusercontent.com/45631812/179449310-002b933e-f444-44be-97d2-1649497f434b.png">

**Comment by Schweinepriester** [comment]
It‘s there, top left :)

<img width="913" alt="image" src="https://user-images.githubusercontent.com/2644614/179473849-c2c5016a-0130-4013-bd1f-240dc1b4bb47.png">

**Comment by OmarHamdann** [comment, marked as answer]
I need Galaxy Brain achievement pls accept my answers.
It already exists, in the top left.

**Reply by Crushoverride007** [reply]
My brain couldn't process that 😆

**Comment by faintout** [comment]
It already exists, the top left is
please each other
https://github.com/faintout/ThreeRoomDemo
